docs/src/documentation/03-components/09-text/heading/03-accessibility.mdx

```diff
@@ -4,25 +4,25 @@ redirect_from:
   - /components/heading/accessibility/
 ---
 
-# Accessibility
-
-## Heading
+## Accessibility
 
 The Heading component has been designed with accessibility in mind.
 
 The component offers flexibility in terms of the HTML element used for the root node, the `role` attribute, and the `level` attribute.
 These properties allow for the creation of semantic and accessible headings.
 
-| Name  | Type                                                    | Description                                                                                                                 |
-| :---- | :------------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------- |
-| as    | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6" \| "div"` | Defines the HTML element to be rendered.                                                                                    |
-| role  | `"heading"`                                             | Can only be used if `as="div"`. If defined, sets the role of the element to be "heading".                                   |
-| level | `number`                                                | Can only be used if `as="div"` and "`role="heading`". Defines the `aria-level` of the rendered `div` with the heading role. |
+| Name  | Type                                                    | Description                                                                                                                |
+| :---- | :------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------------------- |
+| as    | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6" \| "div"` | Defines the HTML element to be rendered as root node.                                                                      |
+| role  | `"heading"`                                             | Can only be used if `as="div"`. If defined, sets the role of the element to be "heading".                                  |
+| level | `number`                                                | Can only be used if `as="div"` and `role="heading"`. Defines the `aria-level` of the rendered `div` with the heading role. |
 
 All the props above are optional by default.
 
-If they are not provided, the component will render a `div` element with no role or aria-level defined.
-It is not semantically wrong but won't tell screen readers that the element is a heading. This should be used only for decorative purposes.
+### Example 1
+
+If the `as` prop is not provided, the component will render a `div` element as a root node. You can optionally provide prop `as="div"` to make it explicit.
```

Comment:
```suggestion
If the `as` prop is not provided, the component will render a `div` element.
```

Per what I said above, and based on what is said below, I think this is more straight to the point. No need to be redundant and say that you can pass the default value as a value, as that is real with every prop in React 😄